### ISSUE TITLE
 ModuleNotFoundError: No module named 'super_gradients'

### ISSUE THREAD

**Issue by louwng** [augmentedstartups/AS-One#45]
![Screenshot_20230608_152037_Chrome](https://github.com/augmentedstartups/AS-One/assets/20241381/1a61af70-a2b9-481a-8cc5-1a09107a8b63)

I get this error when trying to run it in colab

regards
Nico

**Comment by Kush3007** [comment]
I got the same error on cmd, python console and google collab. I tried several times, i thought I was making some mistake. There is also one more issue with the "keras" support, it asks for older version when I install older version it shows the same error.

![asone-yolo error](https://github.com/augmentedstartups/AS-One/assets/67840761/43c7488c-e301-4fea-9306-2aec9988ccd4)

**Comment by mylesalexa** [comment]
I have the same issue, I have tried multiple work arounds,  even when trying to run the sample video there was alot of pip dependencies not installed.  

is there a fix??

**Comment by RobertLowe2** [comment]
I also get this error (on colab).

**Comment by mylesalexa** [comment]
> I also get this error (on colab).

Yes, there might be an issue with the super_gradients repo, Ritesh is looking into this, super_gradients repo causing issues with pycocotools

**Comment by RobertLowe2** [comment]
[like]  Robert Lowe reacted to your message:
________________________________

**Comment by mosdeo** [comment]
same issue, I am M2(Apple Silicon)

**Comment by shersoni610** [comment]
Any solution? I am unable to install this software on a linux machine

**Comment by shersoni610** [comment]
I get the following errors: note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for onnx
  Running setup.py clean for onnx
Failed to build pycocotools onnx
ERROR: Could not build wheels for pycocotools, onnx, which is required to install pyproject.toml-based projects
(asoneenv) %
